dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
+          DependsOnTargets="FilterProjects">
+        <MSBuild Targets="ProducesPackageId"
+             Projects="@(Project)"
+             BuildInParallel="true"
+             ContinueOnError="ErrorAndContinue">
+          <Output TaskParameter="TargetOutputs"
+                  ItemName="PackageIds" />             
+        </MSBuild>
+    <ItemGroup>
+      <PackageIds Include="@(PackageIds)" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="ProducesTarget" DependsOnTargets="FilterProjects">
```

Comment:
Completely understandable.  I'd like to know a bit more about that issue though and why we can't work around it somehow.  It's a bit odd that we now have a third party library telling us what our naming requirements are for our build.